### DISCUSSION TITLE
how to do high availability for celery redbeat scheduled task?

### DISCUSSION THREAD

**Discussion by michaelxu1107** [celery/celery#9468] [Q&A] [answered]
I have two redis proxy, normally i used first redis proxy for celery redbeat, is there any way to change connection automatically to second redis proxy when first redis proxy is down. i have modified redbeat_redis_url to contain both two redis proxies, but it doesn't work. thanks

for celery worker, i configured broker url to contain both two redis proxies, it does work, but celery redbeat does not work

**Comment by auvipy** [comment, marked as answer]
you should ask this to the readbeat repo maintainers
